cmd/util/cmd/common/flow_client.go

```diff
@@ -1,24 +1,60 @@
 package common
 
 import (
+	"fmt"
+	"strings"
+
 	"google.golang.org/grpc"
 
 	"github.com/onflow/flow-go-sdk/client"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/flow/filter"
+	"github.com/onflow/flow-go/state/protocol"
 	"github.com/onflow/flow-go/utils/grpcutils"
+)
 
-	"fmt"
+const (
+	DefaultAccessNodeIDSMinimum = 2
+	DefaultAccessAPIPort        = "9000"
+	DefaultAccessAPISecurePort  = "9001"
 )
 
-// SecureFlowClient creates a flow client with secured GRPC connection
-func SecureFlowClient(accessAddress, accessApiNodePubKey string) (*client.Client, error) {
+type FlowClientOpt struct {
```

Comment:
I would call this `FlowClientConfig` since they are configs instead of options

Reply:
https://github.com/onflow/flow-go/pull/1395/commits/e925198eeb28697d7684f2f86a848e5845a6ebbf